portal/client/src/components/ListView/ListView.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const ListView = ({studyList = [], handleClick}) => {
+    return (
+        <div className="col-4">
+            <div className="list-group">
+                { studyList.map((data, index) => {
+                    if(data) {
+                        return (
+                            <button
+                                key={data.study_id}
+                                className="list-group-item list-group-item-action list-group-item-light"
+                                value={data._id}
```

Comment:
What is the different between `study_id` and `_id` in the `data` object? can you paster sample `data` object here